src/components/explorer/ExplorerSearchInput.tsx

```diff
@@ -4,40 +4,25 @@ import { cn, wait } from 'utils/helpers';
 
 interface Props extends ExplorerSearchProps {
   setShowSuggestions: Dispatch<SetStateAction<boolean>>;
-  onSearchHandler: (v?: string) => void;
   isError?: boolean;
 }
 
 export const ExplorerSearchInput: FC<Props> = (props) => {
-  const placeholder =
+  const label =
     props.type === 'wallet'
       ? 'Search by wallet address'
       : 'Search by token pair';
 
   const onKeyDownHandler = (e: KeyboardEvent<HTMLInputElement>) => {
-    if (e.key === 'Escape') {
-      return props.setSearch('');
-    }
-    if (e.key !== 'Enter') {
-      return;
-    }
-    if (!props.isError && props.search.length > 0) {
-      if (props.type === 'token-pair' && props.filteredPairs.length > 0) {
-        const slug =
-          `${props.filteredPairs[0].baseToken.symbol}-${props.filteredPairs[0].quoteToken.symbol}`.toLowerCase();
-        props.setShowSuggestions(false);
```

Comment:
removing line 24 to 29 introduces the following regressions:

- on enter the first suggestion is not selected
- on enter the suggestions don't disappear